resources/build.js

```diff
@@ -0,0 +1,45 @@
+var execSync = require('child_process').execSync;
```

Comment:
Just a thought: you could use ES6 features in this file if you executed it with `babel-node` instead of `node`.